virttest/qemu_vm.py

```diff
@@ -711,6 +719,8 @@ def add_net(devices, vlan, nettype, ifname=None, tftp=None,
                         if add_vhostfd:
                             cmd += ",vhostfd=%(vhostfd)s"
                             cmd_nd += ",vhostfd=%(vhostfd)s"
+                    if poll_us > 1:
```

Comment:
What if `poll_us=0`? Since you define the default value to `-1`, or you want it default to `0`?

Reply:
poll-us should be a uint32, so set it as -1 as default.